packages/cli/src/util/params.ts

```diff
@@ -45,7 +45,7 @@ export function parseSettings(values: string[] = []) {
 }
 
 export function parsePackageArguments(val: string, result: PackageDefinition) {
-  const packageMatch = val.match(packageRegExp);
+  const packageMatch = val.toLowerCase().match(packageRegExp);
```

Comment:
intention here is to force all package name references to lower case, but this probably isn't the best way to do this...

Reply:
Yeah I was thinking the latter. Maybe we just throw an error if you try to build a package with a name that isn't lower case instead of this?

Reply:
We are acutally already throwing an error on line 50 if the package name is invalid :)